app/models/product.rb

```diff
@@ -64,6 +64,7 @@ class Product < ActiveRecord::Base
   scope :not_archived, -> { where(is_archived: false) }
   scope :mergeable_into_order, -> { not_archived.where(type: mergeable_types) }
   scope :in_active_facility, -> { joins(:facility).where(facilities: { is_active: true }) }
+  scope :recently_purchased, -> { joins(order_details: :order).order('ordered_at DESC').distinct.limit(10) }
```

Comment:
Does `.order(ordered_at: :desc)` work here? If so, it would be better to use that format to take advantage of ActiveRecord.

Reply:
That syntax results in a  `Unknown column 'products.orders'` error, but I think I figured out how to do it with `.merge`